src/styles/main.scss

```diff
@@ -0,0 +1,87 @@
+$main-color: #eee;
+$hover-color: #ffbfcb;
+$font-family: Arial, Halvetica, san-serif;
```

Comment:
There is a typo in the font-family declaration. 'Halvetica' should be 'Helvetica'. Typos in font-family names can lead to unexpected results as the browser may not be able to find the specified font.